app/main.py

```diff
@@ -2,0 +1,10 @@
+def copy_file(command: str) -> None:
+    if command.split() != 3:
+        raise ValueError("Wrong copy command input")
+    cmd = command.split()[0]
+    data_file = command.split()[1]
+    data_file_copy = command.split()[2]
+    if cmd == "cp" and data_file_copy != data_file_copy:
```

Comment:
The condition `data_file_copy != data_file_copy` will always evaluate to False. This seems to be a mistake. You likely intended to check if `data_file_copy` is not equal to `data_file`.